vaadinfrontend/src/main/java/life/qbic/datamanager/views/support/experiment/ExperimentItemCollection.java

```diff
@@ -136,10 +131,8 @@ public void addCreateEventListener(ComponentEventListener<CreationClickedEvent>
   @Override
   public void removeAll() {
     super.removeAll();
-    addCreationCard(createExperiment);
+    addComponentAsLast(createExperiment);
```

Comment:
why did you remove the listener deletion?

Reply:
It's unclear to me why we need to remove the listeners. 
Especially since removing the createListener robs the component of it's functionality. 🤔